Added link to track2 samples in README file (#16929)
Azure/azure-sdk-for-go: sdk/resourcemanager/compute/armcompute/README.md

client := armcompute.NewLogAnalyticsClient(<subscription ID>, cred, &options)
```

## Samples

More code samples for using the management library for Go SDK can be found in the following locations.

> - [Compute Samples](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/new-version/compute)

Reviewer: ## More sample code

- [Availability Set](https://aka.ms/azsdk/go/mgmt/samples?path=compute/availabilityset)
- [Virtual Machine](https://aka.ms/azsdk/go/mgmt/samples?path=compute/createVM)
- [Dedicated Host](https://aka.ms/azsdk/go/mgmt/samples?path=compute/dedicated_host)
- [Disk](https://aka.ms/azsdk/go/mgmt/samples?path=compute/disk)
- [Gallery](https://aka.ms/azsdk/go/mgmt/samples?path=compute/gallery)
- [Proximity](https://aka.ms/azsdk/go/mgmt/samples?path=compute/proximity)
- [Snapshot](https://aka.ms/azsdk/go/mgmt/samples?path=compute/snapshot)
- [Scale Set](https://aka.ms/azsdk/go/mgmt/samples?path=compute/vmscaleset)

> More code samples for using the management library for Go SDK can be found in the following locations.
> - [Compute Samples](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/new-version/compute)
> 

Reviewer: ```suggestion
More code samples for using the management library for Go SDK can be found [here](https://aka.ms/azsdk/go/mgmt/samples)
```

client := armcompute.NewLogAnalyticsClient(<subscription ID>, cred, &options)
```

## More sample code

- [Availability Set](https://aka.ms/azsdk/go/mgmt/samples?path=compute/availabilityset)
- [Virtual Machine](https://aka.ms/azsdk/go/mgmt/samples?path=compute/createVM)
- [Dedicated Host](https://aka.ms/azsdk/go/mgmt/samples?path=compute/dedicated_host)
- [Disk](https://aka.ms/azsdk/go/mgmt/samples?path=compute/disk)
- [Gallery](https://aka.ms/azsdk/go/mgmt/samples?path=compute/gallery)

> - [Proximity](https://aka.ms/azsdk/go/mgmt/samples?path=compute/proximity)

Reviewer 2: ```suggestion
- [Proximity Placement Group](https://aka.ms/azsdk/go/mgmt/samples?path=compute/proximity)
```

- [Snapshot](https://aka.ms/azsdk/go/mgmt/samples?path=compute/snapshot)

> - [Scale Set](https://aka.ms/azsdk/go/mgmt/samples?path=compute/vmscaleset)

Reviewer 2: ```suggestion
- [Virtual Machine Scale Set](https://aka.ms/azsdk/go/mgmt/samples?path=compute/vmscaleset)
```
